content/docs/client-options.md

```diff
@@ -44,6 +44,7 @@ These clients are compatible with our [staging endpoint for ACME v2](https://com
 - [ocelotconsulting/node-acme-lambda](https://github.com/ocelotconsulting/node-acme-lambda)
 - [SSL for free](https://www.sslforfree.com/)
 - [Caddy](https://caddyserver.com) (Caddy >= 0.10.12)
+- [ZeroSSL](https://ZeroSSL.com/) (Online + downloads)
```

Comment:
So far we haven't been qualifying ACMEv2 entries with their particulars except to remark on how to get the V2 compatible code (e.g. if its in a separate branch).

Can you update this line to just add the ZeroSSL link?

Thanks!

Reply:
Sure, no problem. Reduced that to just a link.